integration/tests/tracer/test_tracer_debug_methods.py

```diff
@@ -254,4 +257,183 @@ def test_debug_trace_block_by_non_existent_hash(self):
+                       timeout_sec=120)
+        
+        response = self.tracer_api.send_rpc(
+            method="debug_getModifiedAccountsByNumber", params=[hex(receipt_start["blockNumber"]), hex(receipt_end["blockNumber"])])
+        self.check_modified_accounts_response(response)
+    
+    @pytest.mark.skip(reason="bug NDEV-2375")
+    def test_debug_get_modified_accounts_by_same_number(self):
+        receipt = self.send_neon(
+            self.sender_account, self.recipient_account, 0.1)
+        assert receipt["status"] == 1
+
+        wait_condition(lambda: self.tracer_api.send_rpc(method="debug_getModifiedAccountsByNumber",
+                                                        params=[hex(receipt["blockNumber"]), hex(receipt["blockNumber"])])["result"] is not None,
+                       timeout_sec=120)
+        
+        response = self.tracer_api.send_rpc(
+            method="debug_getModifiedAccountsByNumber", params=[hex(receipt["blockNumber"]), hex(receipt["blockNumber"])])
+        self.check_modified_accounts_response(response)
+   
+    @pytest.mark.skip(reason="bug NDEV-2375")
+    def test_debug_get_modified_accounts_by_only_one_number(self):
+        receipt = self.send_neon(
+            self.sender_account, self.recipient_account, 0.1)
+        assert receipt["status"] == 1
+
+        wait_condition(lambda: self.tracer_api.send_rpc(method="debug_getModifiedAccountsByNumber",
+                                                        params=[hex(receipt["blockNumber"])])["result"] is not None,
+                       timeout_sec=120)
+        
+        response = self.tracer_api.send_rpc(
+            method="debug_getModifiedAccountsByNumber", params=[hex(receipt["blockNumber"])])
+        self.check_modified_accounts_response(response)
+    
+    @pytest.mark.skip(reason="bug NDEV-2375")
+    @pytest.mark.parametrize("difference", [1, 50, 199, 200])
+    def test_debug_get_modified_accounts_by_number_blocks_difference_less_or_equal_200(self, difference):
+        receipt = self.send_neon(
+            self.sender_account, self.recipient_account, 0.1)
+        assert receipt["status"] == 1
+        start_number = hex(receipt["blockNumber"] - difference)
+        end_number= hex(receipt["blockNumber"])
+        wait_condition(lambda: self.tracer_api.send_rpc(method="debug_getModifiedAccountsByNumber",
+                                                        params=[start_number, end_number])["result"] is not None,
+                       timeout_sec=120)
+        
+        response = self.tracer_api.send_rpc(
+            method="debug_getModifiedAccountsByNumber", params=[start_number, end_number])
+        self.check_modified_accounts_response(response)
+    
+    @pytest.mark.skip(reason="bug NDEV-2375")
+    def test_debug_get_modified_accounts_by_number_201_blocks_difference(self):
```

Comment:
why 201 ?

Reply:
cause allowed range of blocks is 200
it is negative case